change/@fluentui-common-styles-2020-07-21-18-01-09-styles-package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "patch",
```

Comment:
```suggestion
  "type": "minor",
```

Reply:
It's starting at v0.1.0 now, so `patch` will make the released version v0.1.1. `minor` would increment it to start at v0.2.0, right?

Reply:
Right. Minor would generally be correct since you're adding new APIs, but practically I don't think it matters in this case.